server/ugRedisApi.js

```diff
@@ -1,107 +0,0 @@
-const redis = require('kth-node-redis')
-const log = require('@kth/log')
-const { server: serverConfig } = require('./configuration')
-
-const redisKeys = (courseCode, semester, ladokRoundIds) => ({
-  teachers: ladokRoundIds.map(round => `${courseCode}.${semester}.${round}.teachers`),
-  examiner: [`${courseCode}.examiner`],
-  responsibles: ladokRoundIds.map(round => `${courseCode}.${semester}.${round}.courseresponsible`),
-  assistants: ladokRoundIds.map(round => `${courseCode}.${semester}.${round}.assistants`), // edu.courses.SF.SF1624.20191.1.assistants
-})
-
-const _removeDublicates = personListWithDublicates =>
```

Comment:
If I remember correctly this function is removing duplicates, f.e., a user created one memo with 3 merged rounds. And there is an assistant LLL which is an assistant in each round, which means we have 3 copies of the same assistant, so it will show 3 times the assistant under the assistant title. Before removing this function it needs to be sure if it handles by the new API or if we still need to remove duplicates.

Reply:
Added remove duplicates method to avoid copy of employee data.